extension/manifest.json

```diff
@@ -9,14 +9,12 @@
     "default_icon": "icon.png",
     "default_popup": "popup.html"
   },
-    "icons":{
-        "64" : "icon.png"
-    },
-    "content_scripts": [
-      {
-         "matches": ["https://www.facebook.com/*"],
-         "css": ["/ladda/ladda.min.css"],
-         "js": [ "myScript.js", "/ladda/ladda.min.js", "/ladda/ladda.min.js"]
-      }
-    ]
+  "icons": {
+    "64": "icon.png"
+  },
+  "content_scripts": [{
+    "matches": ["https://www.facebook.com/*"],
+    "css": ["/ladda/ladda.min.css"],
+    "js": ["myScript.js", "/ladda/ladda.min.js", "/ladda/spin.min.js"]
```

Comment:
@anantdgoel Did you notice that the original line has duplicate files? (ladda.min.js)

`"js": [ "myScript.js", "/ladda/ladda.min.js", "/ladda/ladda.min.js"]`